AEPCampaign/Sources/CampaignConstants.swift

```diff
@@ -118,11 +118,14 @@ enum CampaignConstants {
         static let MESSAGE_CLICKED = "a.message.clicked"
         static let MESSAGE_VIEWED = "a.message.viewed"
         static let MESSAGE_ID = "a.message.id"
+        static let MESSAGE_ACTION_VALUE = "1"
```

Comment:
Should we rename this to `MESSAGE_ACTION_EXISTS_VALUE`? This is to avoid confusion with the Campaign specific message actions (e.g. viewed).

Reply:
yup that is better. will make the update then merge.